### PR TITLE
incorporate a docs type in the tracking board

### PR DESCRIPTION
**Pull request by salehsedghpour**
#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:
This PR adds required documentation for Enhancements Team to incorporate related types and status for docs team in the tracking board.

#### Which issue(s) this PR fixes:
None

#### Special notes for your reviewer:
@gracenng @drewhagen @katcosgrove

### CODE REVIEW

File: release-team/role-handbooks/enhancements/README.md [hunk 1]
```diff
@@ -126,6 +126,7 @@ It is important that this process be followed and documentation remain up-to-dat
       - @release-team-enhancements
   - Create all the Fields (columns) for the board
     > Note: This is currently a manual process and  <https://github.com/orgs/community/discussions/41133> filed to help automate the process.
+    > Note: Since `v1.30` an exceptions process for the Release Doc team is enfoced. The previous PR Ready for Review deadline has been replaced with a Docs Freeze phase. Make sure to add `At Risk for Docs Freeze` and `Tracked for Docs Freeze` to the options in `status` column. Moreover add `docs` to the options in `type` column.
```
Comment (cpanato) [inline release-team/role-handbooks/enhancements/README.md:129]: s/enfoced/enforced

Comment (salehsedghpour) [reply release-team/role-handbooks/enhancements/README.md]: Thanks, done

**Comment by katcosgrove** [comment]
Thanks for doing this!

/lgtm

/assign @rhockenbury

**Comment by drewhagen** [comment]
Thanks!!
cc: @Princesso

**Comment by salehsedghpour** [comment]
Just a kind reminder for having a review on this! Thanks!

**Comment by Priyankasaggu11929** [comment]
@sreeram-venkitesh, could you please take a look at this (if ready to merge)? Thanks!

/approve

/hold 
(unhold, if ready to merge)

**Pull request merged**